app/tests/algorithms_tests/test_views.py

```diff
@@ -312,7 +313,7 @@ def test_algorithm_jobs_list_view(client):
             "length": 50,
             "draw": 1,
             "search[value]": job.creator.username,
-            "order[0][column]": 0,
+            "order[0][column]": JobsList.default_sort_column,
```

Comment:
You've gone from an explicit test, to one depending on your code. What if the content of `JobsList.default_sort_column` does not match the API of what jquery tables expects? That is what this test was testing.

Reply:
I think I see what you mean with the `"desc'` vs `"asc"`, those should likely be explicit.

The other ones: I don't see why those can't directly pull from the code. Since these are meant to mimic the initial or subsequent queries from the page once loaded.

Currently, if the `JobList.default_sort_column` is set to a non-valid value, the tests will fail whereas previously you'd only know if one would load the page.

So unless the test explicitly tests the ordering I'd argue it is fine to short-circuit the definitions to the mock request.